docker/cassandra-Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM cassandra:2.1.22
+FROM cassandra:2.2.19
 COPY ./cassandra.yaml /etc/cassandra/
-CMD ["cassandra", "-f"]
+CMD ["cassandra", "-f", "-Dcassandra.ignore_rack=true"]
```

Comment:
out of curiosity, what's the purpose of this change?

Reply:
Local docker compose for Cassandra fails with `Cannot start node if snitch's rack (rack1) differs from previous rack (RAC1). Please fix the snitch configuration, decommission and rebootstrap this node or use the flag -Dcassandra.ignore_rack=true.`  Adding the above change fixed the issue.

Reply:
that's weird that rack was changed, that's probably related to you local env

Reply:
Thanks, ran docker from the scratch. Don't see that error anymore in local. Reverting the change